composer.json

```
@@ -13,8 +13,8 @@
 			"email": "iqmass@gmail.com"
 		}],
 	"require": {
-		"php": "^8.0",
-		"laravel/nova": "^4.0"
+		"php": "^8",
```

I would like to keep it at the `^8.0`. Technically it should be the same, but I just find it weird to see defined without a minor version :D 
```suggestion
		"php": "^8.0",
```